### PR TITLE
refactor(common): Remove filter-box class

### PR DESCRIPTION
**Pull request by glynnis**
JIRA: https://jira.rax.io/browse/FRMW-219

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM
- [x] QE LGTM

### CODE REVIEW

File: src/components/rxFloatingHeader/scripts/rxFloatingHeader.spec.js [hunk 1]
```diff
@@ -79,7 +79,6 @@ describe('rxFloatingHeader', function () {
         expect(ths.eq(2).hasClass('filter-header'), 'third th').to.be.false;
 
         expect(inputs, 'one input').to.have.length(1);
-        expect(inputs.eq(0).hasClass('filter-box')).to.be.true;
```
Comment (halkon) [inline src/components/rxFloatingHeader/scripts/rxFloatingHeader.spec.js]: I know we talked about this and I said to go ahead and remove it, but if the `rxSearchBox` still applies here, maybe we ought to check for it?

I don't know though, still seems like an unnecessary flimsy test

**Comment by halkon** [comment]
LGTM

File: src/components/rxFloatingHeader/scripts/rxFloatingHeader.spec.js (hunk 1 above)
Comment (CITguy) [inline src/components/rxFloatingHeader/scripts/rxFloatingHeader.spec.js]: I'm okay with this. The logic originally assumed that the first input field in a floating header would be a search box. However, with more advanced filtering patterns emerging, this assumption is becoming less reliable and can lead to unexpected styling problems. The `rxSearchBox` directive isn't added automatically by `rxFloatingHeader` so there's no need to check for it.

**Comment by CITguy** [comment]
LGTM

**Pull request merged**